cmd/install.go

```diff
@@ -278,378 +272,427 @@ func runInstallDupeDetectionSubCommand(ctx context.Context, config *configs.Conf
+		if config.WorkingDir != config.PastelExecDir {
+			log.WithContext(ctx).WithError(err).Errorf("Failed to create folder %s", config.WorkingDir)
 			return err
 		}
-		log.WithContext(ctx).Info(out)
 	}
 
-	return nil
-}
-
-func installChrome(ctx context.Context, config *configs.Config) (err error) {
-	if utils.GetOS() == constants.Linux {
-		log.WithContext(ctx).Infof("Downloading Chrome to install: %s \n", constants.ChromeDownloadURL[utils.GetOS()])
-
-		err = utils.DownloadFile(ctx, filepath.Join(config.PastelExecDir, constants.ChromeExecFileName[utils.GetOS()]), constants.ChromeDownloadURL[utils.GetOS()])
-		if err != nil {
-			return err
-		}
+	config.RPCPort = "9932"
+	if config.Network == "testnet" {
+		config.RPCPort = "19932"
+	}
+	config.RPCUser = utils.GenerateRandomString(8)
+	config.RPCPwd = utils.GenerateRandomString(15)
 
-		if _, err = RunCMD("chmod", "777",
-			filepath.Join(config.PastelExecDir, constants.ChromeExecFileName[utils.GetOS()])); err != nil {
-			log.WithContext(ctx).Error("Failed to make chrome-install as executable")
-			return err
-		}
+	// create pastel.conf file
+	f, err := utils.CreateFile(ctx, config.WorkingDir+"/pastel.conf", config.Force)
+	if err != nil {
+		log.WithContext(ctx).WithError(err).Errorf("Failed to create %s/pastel.conf", config.WorkingDir)
+		return err
+	}
 
-		log.WithContext(ctx).Infof("Installing Chrome : %s \n", filepath.Join(config.PastelExecDir, constants.ChromeExecFileName[utils.GetOS()]))
+	// write to file
+	if err = updatePastelConfigFile(ctx, f, config); err != nil {
+		log.WithContext(ctx).WithError(err).Errorf("Failed to update %s/pastel.conf", config.WorkingDir)
+		return err
+	}
 
-		RunCMDWithInteractive("sudo", "dpkg", "-i", filepath.Join(config.PastelExecDir, constants.ChromeExecFileName[utils.GetOS()]))
+	// create zksnark parameters path
+	if err := utils.CreateFolder(ctx, config.Configurer.DefaultZksnarkDir(), config.Force); err != nil {
+		log.WithContext(ctx).WithError(err).Errorf("Failed to update folder %s", config.Configurer.DefaultZksnarkDir())
+		return err
+	}
 
-		utils.DeleteFile(filepath.Join(config.PastelExecDir, constants.ChromeExecFileName[utils.GetOS()]))
+	// download zksnark params
+	if err := downloadZksnarkParams(ctx, config.Configurer.DefaultZksnarkDir(), config.Force); err != nil &&
+		!(os.IsExist(err) && !config.Force) {
+		log.WithContext(ctx).WithError(err).Errorf("Failed to download Zksnark parameters into folder %s", config.Configurer.DefaultZksnarkDir())
+		return err
 	}
+
 	return nil
 }
 
-func installPackages(ctx context.Context) (err error) {
-	if utils.GetOS() == constants.Linux {
-		installedCmd := utils.GetInstalledCommand(ctx)
-		var notInstall []string
-		for _, p := range constants.DependenciesPackages {
-			if _, ok := installedCmd[p]; !ok {
-				notInstall = append(notInstall, p)
-			}
-		}
+func updatePastelConfigFile(ctx context.Context, fileName string, config *configs.Config) error {
```

Comment:
nit: it's better to use  bytes.Buffer to join lines together before write all to file.